app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -225,16 +225,19 @@ export default class TidyTree {
       this.options.onCollapsedStateChange(node);
   }
 
-  expandCollapsedWithFewChildren(node) {
+  expandCollapsedWithFewChildrenOrNoName(node) {
```

Comment:
I'm not able to expand an "Uncategorized" node after collapsing it (although I checked the old version of the taxon tree and it also behaves poorly there.)

Example on sample 13423:

![image](https://user-images.githubusercontent.com/53838890/70186936-e114bd80-16a1-11ea-9121-ad72ac479c59.png)

![image](https://user-images.githubusercontent.com/53838890/70186931-dd813680-16a1-11ea-8191-fbad4d233e26.png)

It seems to happen with some other nodes as well, but I was able to replicate it most consistently with the "Uncategorized" ones.

Reply:
Fixed.